helmfile/helmfile-infra.yaml

```diff
@@ -40,6 +40,9 @@ releases:
   chart: kubernetes-dashboard/kubernetes-dashboard
   version: ~4.3.1
 
+- name: kubedashboard-admin-sa
+  chart: ../charts/kubedashboard-admin-sa
```

Comment:
WDYT if we combine kubedashboard-admin-sa and kubernetes-dashboard to one chart and define kubernetes-dashboard as a dependency?

Reply:
It's not bad idea, but i believe that it's not needed, cause I'm going to add ability to disable each component and on most of clouds dashboard could be used without that SA